src/FrontendObligationChecker/ISharedResources.cs

```diff
@@ -1,5 +1,5 @@
 ﻿namespace FrontendObligationChecker;
 
-public class SharedResources
+public interface ISharedResources
```

Comment:
can we not just delete this empty class?

Reply:
updated